### ISSUE TITLE
Conv2d_0/BatchNorm/cond was passed float from .../Conv2d_0/BatchNorm/gamma:0 incompatible with expected resource

### ISSUE THREAD

**Issue by manuelcosta74** [tf-coreml/tf-coreml#385]
## 🐞Describe the bug
Trying to convert a model created with TF 2.1 and having the following error when minimum_ios_deployment_target = '13':

Conv2d_0/BatchNorm/cond was passed float from .../Conv2d_0/BatchNorm/gamma:0 incompatible with expected resource

If the default minimum_ios_deployment_target is used it triggers the following error 

module 'tensorflow' has no attribute 'reset_default_graph' 

## To Reproduce
- If a python script can reproduce the error, please paste the code snippet
```
import tfcoreml
import coremltools
import copy
import tensorflow as tf
import argparse

from coremltools.proto import FeatureTypes_pb2 as ft
from  coremltools.models.neural_network.printer import print_network_spec

def parse_args():
    parser = argparse.ArgumentParser(
        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
        description='data path information'
    )

    parser.add_argument("--tf_model_path", type=str,
                        help='path to the Tensor Flow Model')
    parser.add_argument("--ml_raw_path", type=str,
                        help='path to the CoreML converted file')
    parser.add_argument("--ml_model_path, type=str,
                        help='path to the final CoreML with a l2normalize layer in the end')
                        
    
    
    args = parser.parse_args()
    return args

if __name__ == '__main__':
    args = parse_args()

    # TF graph definition
    
    tf_model_path = args.tf_model_path

    # Output CoreML model path
    coreml_model_file = args.ml_raw_path
    
    # img = img - 127.5
    # img = img * 0.0078125

    scale = 1.0 / 128.0

    coreml_model = tfcoreml.convert (tf_model_path=tf_model_path,
                                    mlmodel_path=coreml_model_file,
                                    input_name_shape_dict={"img_inputs": [1, 112, 112, 3]}, # batch size is 1
                                    image_input_names = ['img_inputs'],
                                    output_feature_names= ['embeddings'],
                                    red_bias    = - 127.5 * scale,
                                    green_bias  = - 127.5 * scale,
                                    blue_bias   = - 127.5 * scale,
                                    image_scale = scale)
                                    #,
                                    #sminimum_ios_deployment_target ='13')


    spec = coreml_model.get_spec()

    nn_spec = spec.neuralNetwork


    print("####### 1 #######")
    print(nn_spec.preprocessing)

    input = spec.description.input[0]
    output = spec.description.output[0]

    input.type.imageType.colorSpace = ft.ImageFeatureType.ColorSpace.Value('RGB')


    input.type.imageType.width = 112  
    input.type.imageType.height = 112


    # get NN portion of the spec
    nn_spec = spec.neuralNetwork

    old_name = input.name
    input.name = 'image'
    output.name = 'output'

    for layer in nn_spec.layers:
        for i, inp_ in enumerate(layer.input):
            if inp_ == old_name: #whatever the old input name is
                print("Layer name changed from  " + inp_ + " to " + input.name)
                layer.input[i] = input.name #whatever the new desired input name is
                
    #nn_spec.layers[0].input[0] = 'image'
    nn_spec.layers[len(nn_spec.layers)-1].output[0] = 'output'

    nn_spec.layers[len(nn_spec.layers)-1].output[0] = 'embeddings_raw'
    embeddings_norm_layer = nn_spec.layers.add()
    embeddings_norm_layer.name = 'embeddings_norm_layer'
    embeddings_norm_layer.input.append('embeddings_raw')
    embeddings_norm_layer.output.append('output')

    params =  embeddings_norm_layer.l2normalize
    params.epsilon = 1e-6

    coreml_model = coremltools.models.MLModel(spec)
    spec = coreml_model.get_spec()

    nn_spec = spec.neuralNetwork

    print("####### 2 #######")
    print(nn_spec.preprocessing)

    coremltools.utils.save_spec(spec, args.ml_model_path)


```
pb Model available here : FB7613102 

## System environment (please complete the following information):

pip list on the used python environment

Package              Version   
-------------------- ----------
absl-py              0.9.0     
astor                0.8.1     
cachetools           4.0.0     
certifi              2019.11.28
chardet              3.0.4     
coremltools          3.3       
dlib                 19.18.0   
gast                 0.2.2     
google-auth          1.11.2    
google-auth-oauthlib 0.4.1     
google-pasta         0.1.8     
grpcio               1.27.2    
h5py                 2.10.0    
idna                 2.9       
Keras-Applications   1.0.8     
Keras-Preprocessing  1.1.0     
mac-vlc              2019.6.25 
Markdown             3.2.1     
mpmath               1.1.0     
numpy                1.17.4    
oauthlib             3.1.0     
opencv-python        4.1.2.30  
opt-einsum           3.1.0     
pafy                 0.5.5     
pip                  19.3.1    
protobuf             3.11.3    
pyasn1               0.4.8     
pyasn1-modules       0.2.8     
PyQt5                5.13.2    
PyQt5-sip            12.7.0    
python-vlc           3.0.7110  
requests             2.23.0    
requests-oauthlib    1.3.0     
rsa                  4.0       
scipy                1.4.1     
setuptools           42.0.1    
six                  1.14.0    
sympy                1.5.1     
tensorboard          2.1.0     
tensorflow           2.1.0     
tensorflow-estimator 2.1.0     
termcolor            1.1.0     
tfcoreml             1.1       
urllib3              1.25.8    
Werkzeug             1.0.0     
wheel                0.33.6    
wrapt                1.12.0    
youtube-dl           2019.11.22
 
 
 - macOS version (if applicable): 10.15.4 Beta (19E242d
 - How you install python (anaconda, virtualenv, system): virtualenv
 - python version (e.g. 3.7): 3.7
 - any other relevant information:

## Additional context

This model was converting without any issues before converting tensorflow to 2 through the execution of tf_upgrade_v2 \
  --intree my_project/ \
  --outtree my_project_v2/ \
  --reportfile report.txt
